code-challenges-revised.test.js

```diff
@@ -0,0 +1,158 @@
+// ASSESSMENT 2: Coding Practical Questions with Jest
+
+// Please read all questions thoroughly
+// Pseudo coding is REQUIRED
+// If you get stuck, please leave comments to help us understand your thought process
+
+// Use test driven development to complete the following questions
+// Add appropriate dependencies: $ yarn add jest
+
+// Reminder: The test will call your function
+// Run the file with the following command: $ yarn jest
+
+
+// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+
+// a) Create a test with expect statements for each of the variables provided.
+
+var num1 = 15
+// Expected output: "15 is divisible by three"
+var num2 = 0
+// Expected output: "0 is divisible by three"
+var num3 = -7
+// Expected output: "-7 is not divisible by three"
+
+// 1a) Pseudo code:
+// Create describe, it, and expect statements to describe the problem statement
+// Add appropriate inputs/outputs, with one expect line per input/output pair
+// Run the test on terminal (yarn jest) prior to creating the function to ensure it turns red (note: if set up correctly, should get an error saying the function is not defined)
+
+describe("divisibleByThree", () => {
+    it("a function that takes in a number argument and determines if it is evenly divisible by 3", () => {
+      expect(divisibleByThree(num1)).toEqual("15 is divisible by three")
+      expect(divisibleByThree(num2)).toEqual("0 is divisible by three")
+      expect(divisibleByThree(num3)).toEqual("-7 is not divisible by three")
+    })
+  })
```

Comment:
The test looks good, the only thing it is missing is putting the test variables IN the test.